soroban-env-host/Cargo.toml

```diff
@@ -13,8 +13,8 @@ rust-version = "1.71"
 [dependencies]
 soroban-builtin-sdk-macros = { workspace = true }
 soroban-env-common = { workspace = true, features = ["std", "wasmi"] }
-stellar-strkey = { workspace = true }
 wasmi = { workspace = true }
+stellar-strkey = "0.0.8"
```

Comment:
I had to pin the strkey dependency to crates registry instead of commit hash. Because it was failing cargo-deny showing up as two different version (https://github.com/stellar/rs-soroban-env/actions/runs/6793313451/job/18467959394). `rs-stellar-xdr` is depending on strkey using the crates registry.  
I'm not sure why this previously wasn't an issue though. 
@leighmcculloch let me know if this is not the right move.

Reply:
We shouldn't ever need to use a hash git ref for the stellar-strkey package. It's low level enough that when we make changes to it we ship them immediately as a release. Their releases are not tied to overall system releases.